src/javascripts/ng-admin/Main/config/factories.js

```diff
@@ -1,33 +0,0 @@
-/*global define*/
-
-define(function (require) {
```

Comment:
@manuquentin, @jeromemacias, can you confirm that this file is obsolete with `admin-config`?